gnd/src/main/java/com/google/android/gnd/model/basemap/Area.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.android.gnd.model.basemap;
+
+import com.google.android.gms.maps.model.LatLngBounds;
+import com.google.auto.value.AutoValue;
+
+/**
+ * An area is a contiguous set of tiles that form a geodesic rectangle.
+ */
+@AutoValue
+public abstract class Area {
+  public static Builder newBuilder() {
```

Comment:
Nit: We've been putting `newBuilder` above  the Builder class definition before.

Reply:
Gotcha--whatever formatting settings I have want to push this up to the top. 

I feel like I may still not have the right .idea files or something--are those checked into git?